src/FakeItEasy.Analyzer/NonVirtualSetupAnalyzer.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;

```

Comment:
This is a nitpick, but FakeItEasy style is to put the usings inside the namespace, sorted alphabetically (with System* in a group at the top), like on line 6 and below.

This can be seen in the compilation warnings in the IDE (or in the [CI build](http://teamcity.codebetter.com/viewLog.html?buildId=225343&buildTypeId=bt929&tab=buildLog&state=59%2C243#_state=59,243&focus=258)), along with a number of other warnings that have been introduced.

I point this out not to force you to do more work, @medmondson, but because if I were a contributor, I'd want to know about the target project's conventions. Don't worry too much about the warnings @medmondson. We won't reject the PR based on them. But if you _wanted_ to clean them up, either as part of other rework or not, the effort would be welcome and appreciated.